src/app/yourcity/yourcity.component.ts

```diff
@@ -0,0 +1,60 @@
+import { Component, OnInit } from '@angular/core';
+import { YourcityService } from "../service/yourcity.service";
+
+@Component({
+  selector: 'app-yourcity',
+  templateUrl: './yourcity.component.html',
+  styleUrls: ['./yourcity.component.css']
+})
+export class YourcityComponent {
+  nameCity: any;
+  temp: any;
+  weather: any;
+  timeOfDay: string = 'day';
```

Comment:
use `ENUM` for a different type of value read the comments below about the factory class or switch case and combine it.